trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, a TC represents the interests of both
+their InnerSource community and the products the community is building. So as
+a TC, you'll have both tech oriented and community oriented responsibilities
+and we'll explore both of these dimensions in the following sections.
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor
+    - makes contributions
+    - sometimes part of the community
+    - sometimes send by another part of the organization to develop a feature
+      for it. This is why contributors are sometimes referred to as Guests. 
+    - responsible for "fitting in"/conforming to the host team and it's 
+      expectations and processes
+    - similar to an Apache Developer or committer
+
+- Trusted Committer
+    - Sometimes referred to as a host.
+    - In this analogy, he is both resonsible for building the house and setting
+      the house rules to make sure his guests are comfortable.
+    - In comparison to the Contributor, a TC has rights to push code closer to
```

Comment:
I also think we should not tie the TC role explanation to technical implementations at this point. I like your explanation @cewilliams that a TC is trusted to do more _risky_ stuff than a _Contributor_.

Reply:
On the use of pronouns, instead of "In this analogy, _he_ is both resonsible for building the house and setting..." could we please use "they"? I'm not suggesting that we need to use _she_, just that we be gender neutral.

Reply:
Related to pronouns: During the 10/17 discussion - we agreed to use a combination of they/their and the actual role name, instead of he/she.